src/middlewared/middlewared/etc_files/local/avahi/services/ADISK.service.py

```diff
@@ -0,0 +1,42 @@
+from middlewared.plugins.etc import FileShouldNotExist
+from middlewared.utils import filter_list
```

Comment:
Nit: filter_list is not used.
Here and in the other service files.

Reply:
Good catch.